components/gcp/automl/export_data_to_gcs/component.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import NamedTuple
+
+
+def automl_export_data_to_gcs(
+    dataset_path: str,
+    gcs_output_uri_prefix: str = None,
+    #retry=None, #=google.api_core.gapic_v1.method.DEFAULT,
+    timeout: float = None, #=google.api_core.gapic_v1.method.DEFAULT,
+    metadata: dict = {},
```

Comment:
if possible should add  sub-scripting instead of generic dict

Reply:
Unfortunately, this dict does not have regular structure.